examples/main.tf

```diff
@@ -3,6 +3,11 @@ provider "instaclustr" {
   api_key = "<Your provisioning API key here>"
 }
 
+resource "instaclustr_encryption_key" "add_ebs_key" {
+    alias = "<Your KMS key alias here>"
+    arn = "<Your KMS key ARN here>"
+}
+
 resource "instaclustr_cluster" "example2" {
```

Comment:
disk_encryption_key = "${instaclustr_encryption_key.add_ebs_key.key_id}" could be added here for a better example for disk_encryption_key